README.md

```diff
@@ -3,17 +3,11 @@
 This project is an extension for VS Code to provide the features of the [Checker Framework](https://checkerframework.org/),
 via the [Checker Framework Language Server](https://github.com/eisopux/checker-framework-languageserver).
 
-
 ## Getting Started
 
-You can either install the extension from the
-[VS Code Extension Marketplace](https://code.visualstudio.com/docs/editor/extension-marketplace)
-or install a locally-built version of the extension.
-Please refer to the "Developer's Guide" below for instructions on how to build from source.
+You can either install the extension from the [VS Code Extension Marketplace](https://code.visualstudio.com/docs/editor/extension-marketplace) or install a locally-built version of the extension.Please refer to the "Developer's Guide" below for instructions on how to build from source.
 
-After installing the extension, when you open or save any `.java` file it will be checked
-by the Checker Framework [Nullness Checker](https://checkerframework.org/manual/#nullness-checker).
-Other type systems and options can be enabled in the configuration.
+After installing the extension, when you open or save any `.java` file it will be checked by the Checker Framework [Nullness Checker](https://checkerframework.org/manual/#nullness-checker). Other type systems and options can be enabled in the configuration.
```

Comment:
Same comment about linebreaks here. Did anything change?

Reply:
I think the line breaks were just a formatting error on my part; however, nothing actually changed. Sorry about that. I will fix them right now!